### DISCUSSION TITLE
Google Cast Known Hosts Configuration (mDNS)

### DISCUSSION THREAD

**Discussion by justin-arens** [music-assistant/support#3296] [Feature requests and ideas]
I have a bunch of Google Home Devices that I am currently using with the default Google Cast HA Integration. My isp provided a dumpster of a router that blocks mDNS for some reason. The Google Cast HA Integration offers a "Known hosts" configuration option (Known Hosts - A comma-separated list of hostnames or IP-addresses of cast devices, use if mDNS discovery is not working.). 

Is it possible to add this feature for the Google Cast Player Provider? I am ok with using the Home Assistant Player Provider and absorbing the players from HA, but I see the warning in the doc that suggest there might be a benefit to adding them to MA directly.

**Comment by OzGav** [comment]
Its a fair request but with the load of bugs and improvements it might take while

**Reply by justin-arens** [reply]
Thanks for the quick reply, I understand completely! Obvious solution is to address the root cause and replace the router. Would have done that ages ago if the known hosts config option didn't exist for the Google Cast integration. Just checking out MA for the first time, looks nifty! I will absorb the players and keep poking around, keep up the great work guys!
